specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Storage/Storage.tsp

```diff
@@ -182,6 +182,23 @@ namespace Microsoft.EventGrid.SystemEvents {
     errorList?: string;
   }
 
+  /** The access tier of the blob */
+  union StorageBlobAccessTier {
```

Comment:
Can we add this to the client.tsp under this folder with the @@access and @@usage decorators?
```

@@usage(Microsoft.EventGrid.SystemEvents.StorageBlobAccessTier,
  Usage.output
);
@@access(Microsoft.EventGrid.SystemEvents.StorageBlobAccessTier,
  Access.public
);
```

Reply:
Added to client.tsp